common/JackRequest.h

```diff
@@ -1629,6 +1630,48 @@ struct JackClientHasSessionCallbackRequest : public JackRequest
     
 };
 
+
+struct JackPropertyChangeNotifyRequest : public JackRequest
+{
+    jack_uuid_t fSubject;
+    char fKey[JACK_UUID_STRING_SIZE];
+    jack_property_change_t fChange;
+
+    JackPropertyChangeNotifyRequest()
```

Comment:
fChange is not initialized here

Reply:
fixed on https://github.com/jackaudio/jack2/pull/383/commits/d5a4ba6